src/helpers/firmware/installMcu.js

```diff
@@ -5,6 +5,20 @@ import { WS_MCU } from 'helpers/urls'
 import { createDeviceSocket } from 'helpers/socket'
 import getNextMCU from 'helpers/firmware/getNextMCU'
 import getDeviceInfo from 'helpers/devices/getDeviceInfo'
+import { createCustomErrorClass } from 'helpers/errors'
+
+const ManagerDeviceLockedError = createCustomErrorClass('ManagerDeviceLocked')
+
+function remapSocketError(promise) {
+  return promise.catch((e: Error) => {
+    switch (true) {
+      case e.message.endsWith('6982'):
```

Comment:
(note for future, better way to do it is using the error field: https://github.com/LedgerHQ/ledger-live-desktop/blob/708eaa5e4492a077c58bda766396a34fedbd3d08/src/commands/getAddress.js#L35-L42